### PR TITLE
added supported browsers

### PR DESCRIPTION
**Pull request by Saviour1001**

### CODE REVIEW

File: docs/troubleshooting/supported-browsers.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Supported Browsers List
+image: "/docs/images/docs-meta-cards/troubleshoot-card.png"
+displayed_sidebar: docs
+description: "Webpack 5 Issues | Documentation - Web3Auth"
```
Comment (shahbaz17) [inline docs/troubleshooting/supported-browsers.mdx:5]: @Saviour1001 

the description says it's a `Web3pack 5 Issues`.

File: docs/troubleshooting/supported-browsers.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+---
+
+## Supported Browsers List
+
+Web3Auth SDKs have functions which require BigInt compatability and thus the browsers which are supported are as follows. The browsers are listed
```
Comment (shahbaz17) [inline docs/troubleshooting/supported-browsers.mdx:12]: In this sentence, both the lines convey the same thing, can we merge those together and construct one line?

File: docs/troubleshooting/supported-browsers.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+alongside their versions which are supported.
+
+- Chrome 67+
```
Comment (shahbaz17) [inline docs/troubleshooting/supported-browsers.mdx:15]: Also, what was the browserlist config to get the below result on https://browsersl.ist?

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

File: docs/troubleshooting/supported-browsers.mdx (hunk 2 above)
Comment (yashovardhan) [reply docs/troubleshooting/supported-browsers.mdx:12]: I agree, please remove the H2 and add description properly on why and how this error is happening. Add the link: https://browsersl.ist/#q=supports+bigint%2C+not+dead

File: docs/troubleshooting/supported-browsers.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- Samsung Internet 9.2+
+- UC Browser for Android 13.4+
+- QQ Browser 13.1+
+- Android Browser 107+
+
+:::note source code
+
+When creating your React App in production, make sure to follow the steps described [here](./react-big-int-error) to avoid BigInt Error.
+
+:::
```
Comment (yashovardhan) [inline docs/troubleshooting/supported-browsers.mdx:33]: change this to `:::info source code`

File: docs/troubleshooting/supported-browsers.mdx (hunk 1 above)
Comment (yashovardhan) [reply docs/troubleshooting/supported-browsers.mdx:5]: remove `/docs/` from image (check recent change according to `vercel-fix`) and change the description.

**Review by yashovardhan — CHANGES_REQUESTED** [review]
You doc is fine, but in terms of writing please note that repeated headings cause lower SEO rankings and also bad user experience, please make sure you update according to the comments

**Review by yashovardhan — APPROVED** [review]

**Pull request merged**